Doc/whatsnew/3.15.rst

```diff
@@ -83,7 +83,11 @@ Other language changes
 New modules
 ===========
 
-* None yet.
+intmath
```

Comment:
Maybe add an entry under the PEP-sized items as well?

Reply:
Yes, but for 3.14, I think we're mentioning them all? @hugovk do you want to mention the PEP? Or at least, we should have a link to the PEP. There is none at the moment (`annotationlib` got one)

Reply:
Ask me after the PEP has been accepted :)